library/std/src/thread/mod.rs

```diff
@@ -887,6 +879,23 @@ pub fn sleep(dur: Duration) {
 ///
 /// * It can be implemented very efficiently on many platforms.
 ///
+/// # Memory Orderings
+/// 
```

Comment:
<https://github.com/rust-lang/rust/runs/7461557231?check_suite_focus=true#step:25:2218>:

```
tidy error: /checkout/library/std/src/thread/mod.rs:883: trailing whitespace
```

```suggestion
///
```